po/de.po

```diff
@@ -2,8 +2,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: gsconnect\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-12-18 19:41+0100\n"
-"PO-Revision-Date: 2020-12-18 20:11+0100\n"
+"POT-Creation-Date: 2021-08-07 07:51+0200\n"
```

Comment:
```suggestion
"POT-Creation-Date: 2020-12-18 19:41+0100\n"
```

This field shouldn't be updated according to https://www.gnu.org/software/trans-coord/manual/gnun/html_node/PO-Header.html

Reply:
Yeah, when following the wiki instructions for translating GSConnect, one step is to update the `pot` file. When doing so, this line changes in every `po` file - which is intended.
But since I didn't commit the new `pot` file, you are right! Thanks for noticing!